### PR TITLE
Add Prophecy libs page

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208012876600748

### CODE REVIEW

File: docs/release_notes/prophecy-libs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Out of the box, Spark lacks auditing or monitoring capabilities for its Pipelines. Prophecy Libs bridges this gap by offering step-by-step transformation monitoring, enabling users to easily comprehend Pipeline execution metrics (e.g., times of arrival, amount of data processed), data profiles, and detailed debugging information at every Pipeline step.
+
+:::note
+
+Pipeline Monitoring is fully independent of the Prophecy IDE, using standard metadata tables. Reading and leveraging this information for downstream consumption is as simple as reading a table.
+
+:::
+
+To learn more about Prophecy monitoring capabilities, see [Pipeline Monitoring](/docs/Spark/pipeline-monitoring/pipeline-monitoring.md). And to learn more about the metrics setup, see [Execution Metrics](/docs/Spark/execution/execution-metrics.md).
```
[the remark this answers is not in the document]
Comment (atbida) [reply docs/release_notes/prophecy-libs.md:32]: Perhaps the cluster needs to be attached, per this [Asana.](https://app.asana.com/0/1203813959485797/1208439860760129/f) Testing.

File: docs/release_notes/prophecy-libs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+Prophecy Libs are small and lightweight utility libraries, offering comprehensive sets of Spark functions tailored for Prophecy customers. They address the most common data transformation use cases, including monitoring, auditing, connectors, and other data processing functions. These versatile libraries support Scala and Python Spark environments.
+
+Here are the released artifacts:
+
+- Prophecy Libs for Scala: [Maven](https://mvnrepository.com/artifact/io.prophecy/prophecy-libs)
+- Prophecy Libs for Python: [PyPi](https://pypi.org/project/prophecy-libs/)
+
+For a list of the latest versions, see [Version Chart](/docs/release_notes/version_chart.md).
```
Comment (atbida) [inline docs/release_notes/prophecy-libs.md:16]: How can the user know the libs version installed on the cluster?
(Aside from waiting until the cluster restarts?)

Comment (atbida) [reply docs/release_notes/prophecy-libs.md:16]: Suggest: "IDentify which plibs version is running on your Spark cluster by checking the cluster dropdown in the pipeline."

Comment (alexanderahn) [reply docs/release_notes/prophecy-libs.md]: Done.

**Comment by alexanderahn** [comment]
Got LGTM in Slack https://simpledatalabs.slack.com/archives/C04CQJVSR6J/p1727766669384749?thread_ts=1727757615.177579&cid=C04CQJVSR6J

**Pull request merged**